### ISSUE TITLE
Popup vs. popup with inline block inserter and toolbar block inserter

### ISSUE THREAD

**Issue by haszari** [WordPress/gutenberg#22156]
**Describe the bug**
<img width="882" alt="Screen Shot 2020-05-07 at 3 57 44 PM" src="https://user-images.githubusercontent.com/4167300/81253051-a3b2c280-907b-11ea-954b-6fcbc48e9c9a.png">

In Gutenberg you can add blocks from the toolbar block inserter and also from a keyboard-shortcut when typing a `/`.

It's possible to have both inserters display at the same time, overlapping each other, which is confusing.

**To reproduce**
1. Edit a page or post in block editor.
2. Click in content are and type `/` to bring up inline inserter.
3. Realise you have no clue what text to type to get the block you want, so mouse up to toolbar and click the big plus.

**Expected behavior**
There should only be one popup visible on screen at a time, and it should be focused. Or, the most recently opened one should be fully visible.

**Screenshots**
(Screenshot above)

**Editor version (please complete the following information):**
- WordPress version: 5.4.1
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? `default`

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version: 81.0.4044.129

**Comment by annezazu** [comment]
Thanks for this report @haszari! I was able to replicate this using just 5.4.1. Flagging this for the team! To be safe, I tested this with Gutenberg 8.1 RC (no experiments) and WordPress 5.4.1 only to find what seems to be an even worse duplication happening:

<img width="1169" alt="Screen Shot 2020-05-12 at 11 15 35 AM" src="https://user-images.githubusercontent.com/26996883/81730212-06bca480-9442-11ea-98c4-5091cb2a348e.png">

Here's a video: https://cloudup.com/cPQ3YL6U_Xh

**Comment by youknowriad** [comment]
I think this is invalid now that the inserter shows up as a panel.
